apps/typegpu-docs/src/content/docs/fundamentals/enabling-features.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Enabling GPU Features
+description: How to declare and activate WebGPU device features in TypeGPU.
+---
+
+Some GPU capabilities, such as timestamp queries or subgroup operations, must be explicitly requested before use. TypeGPU passes your configuration directly to the WebGPU API, with a convenient extension for marking features as optional.
+
+:::tip
+For a complete list of feature names, consult the [WebGPU specification](https://www.w3.org/TR/webgpu/#gpufeaturename).
+:::
+
+## Requesting Features
```

Comment:
we've been using Sentence case for the section headers in other docs, let's stay consistent